codebuild/bin/install_default_dependencies.sh

```diff
@@ -13,7 +13,7 @@
 # permissions and limitations under the License.
 #
 
-set -ex
+set -e
```

Comment:
Were the logs getting cluttered with -x set? I kind of liked being able to trace through what it was doing, but maybe we don't need it.